ecosystem/sep-0046.md

```diff
@@ -0,0 +1,321 @@
+## Preamble
+
+```
+SEP: 0046
+Title: Key Sharing Method for Stellar Keys
+Authors: Pamphile Roy <@tupui>
+Track: Standard
+Status: Draft
+Created: 2025-01-15
+Updated: 2025-01-15
+Version 0.1.0
+Discussion: TBA
+```
+
+## Simple Summary
+
+This proposal describe a method to share Stellar Keys. The interface is an extension of the [SEP-5].
+
+## Motivation
+
+Smart wallets are becoming ubiquitous. Most wallet leverage a mnemonic codes which users are
+asked to safely store offline. This poses some operational security concerns and brings the need for
+a standard to store secrets offline in a secure and resilient manner.
+
+Such method should have the following attributes:
+
+- allow to split a secret in shares,
+- guarantee that the secret can only be deciphered using a given number of shares,
+- allow the use of an additional passphrase to further protect the secret.
+
+While similar in some ways, multi-signature is a different concept and has to be considered
+separately and should be viewed as complementary.
+
+Adopting a standard methodology should significantly improve self-custodian setups' security
+giving user more peace of mind.
+
+## Specification
+
+Shamir Secret Sharing (SSS) method, as described in 
+[SLIP-0039](https://github.com/satoshilabs/slips/blob/master/slip-0039.md) should be used as a way to split a
+mnemonic code into multiple parts. SSS is a two-layer scheme: _G_ groups are split into _N_ distinct members.
+_g_ ≤ _G_ groups are required to reconstruct the mnemonic. Each group being comprised of a specific threashold of
+members 1 ≤ _t_ ≤ _N_.
+
+The entropy (which can be derived from a mnemonic code) should be used to derive the multiparts menomonic.
+The parts are non-deterministic and any derivation leads to the same set of keys. As recommended for single
+mnemonic codes, we recommend using 256 bits of entropy. Which leads to 33 word mnemonic phrases.
+
+While BIP-0039 can be derived from the entropy, it is not compatible with
+SLIP-0039 as noted in the standard. This means that BIP-0039 derived addresses
+are different from SLIP-0039 derived addresses.
+
+## Design Rationale
```

Comment:
It would be better to explain the reasons for choosing SLIP-39 here; in my opinion, `Alternatives considered include:` is a bit redundant.

```
SLIP-39 was chosen as the base standard because:

1. Industry adoption and testing
2. Hardware wallet support
3. ...
```

Reply:
I rephrased a bit. Please feel free to add some suggestions.